server/src/db/seeders/ProductSeeder.js

```diff
@@ -0,0 +1,47 @@
+import Product from "../../models/Product.js"
+
+class ProductSeeder {
+  static async seed() {
+    const productsData = [
+      {
+        productName: "Heritage Style Whole Grain Bread Gluten Free",
+        brandName: "Canyon Bakehouse",
+        description: "Certified Gluten Free. Dairy, Soy, and nut free.",
+        categoryId: "1"
```

Comment:
Instead of specifying the id of the category to associate with, a stronger seed file would query for those existing categories and relate the category with the creation of these products